### PR TITLE
Fix is unique nesting

### LINKED ISSUE #67: Correct "is unique" event format
The following are the response variables declared by the "is unique" integration:

```
is_unique.outcome
is_unique.reason
is_unique.query_item.outcome
is_unique.query_item.reason
is_unique.query_item.found
is_unique.query_item.found_in
is_unique.query_item.added_at
is_unique.add_item.outcome
is_unique.add_item.reason
is_unique.add_item.accepted
is_unique.add_item.rejected
```

And here's what the "is unique" event looks like:

```json
{
  "id": "59de867bed8af01fb7f01572",
  "type": "recipient",
  "outcome": "success",
  "reason": null,
  "key": "suppressionlist",
  "vars": {
    "lead": { ... },
    "source": { ... },
    "recipient": { ... },
    "suppressionlist": {
      "query_item": {
        "specified_lists": [
          "tresemme_dedupe"
        ],
        "key": "uniqueesolan@gmail.com",
        "found": {
          "normal": false,
          "valid": true,
          "raw": false
        },
        "outcome": "success",
        "reason": null
      }
    }
  },
  "module_id": "leadconduit-suppressionlist.outbound.is_unique",
  "package_version": "2.0.4",
  "handler_version": "7.8.0",
  "transactions": [ ... ],
  "mappings": [
    {
      "property": "value",
      "value": "{{lead.email}}",
      "id": "93rkCb"
    },
    {
      "property": "list_name",
      "value": "unilever_duping",
      "id": "nCFl4l"
    }
  ],
  "start_timestamp": 1507755643808,
  "end_timestamp": 1507755643877,
  "ms": 69,
  "wait_ms": 49,
  "total_ms": 157,
  "suppressionlist": {
    "query_item": {
      "specified_lists": [
        "unilever_duping"
      ],
      "key": "uniqueesolan@gmail.com",
      "found": false,
      "outcome": "success",
      "reason": null
    },
    "add_item": {
      "accepted": 1,
      "rejected": 0,
      "outcome": "success",
      "reason": null
    },
    "is_unique": {
      "outcome": "success"
    }
  },
  "overhead_ms": 20,
  "lag_ms": 0,
  "expires_at": "2018-01-19T21:00:43.877Z"
}
```

Note that the appended `add_item` and `query_item` properties are not nested under `is_unique` like the response variables claim. That means that rules based on these properties will fail. It also means that data appended during prior calls to "add_item" or "query_item" will be overwritten by calls to "is_unique" (such is the case with this event).

### PR DESCRIPTION
**Pull request by chasingSublimity**
closes #67.

### CODE REVIEW

**Comment by chasingSublimity** [comment]
Heard.

**Comment by chasingSublimity** [comment]
@cgrayson round 2.

NB: There's currently no test coverage for the duplicated data. Happy to add some if you deem it necessary.

File: src/is-unique.js [hunk 1]
```diff
@@ -45,22 +45,25 @@ const handle = (vars, callback) => {
 
   query(queryVars, (err, queryEvent) => {
     if (err) return callback(err);
-    const event = _.merge({}, queryEvent);
+    // Duplicating data here until affected flows are corrected
+    const event = _.merge({is_unique: {}}, queryEvent);
+    _.merge(event.is_unique, queryEvent);
+
+    console.log('event1: \n', event)
```
Comment (cgrayson) [inline src/is-unique.js]: Oops here (and L62) :-)

Comment (chasingSublimity) [reply src/is-unique.js]: gah! I need to write a hook that searches for those, haha.

**Pull request merged**